sycl/unittests/pi/CMakeLists.txt

```diff
@@ -2,11 +2,6 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-# Enable exception handling for these unit tests
-set(LLVM_REQUIRES_EH 1)
-add_llvm_unittest(PiTests
+add_sycl_unittest(PiTests
   PlatformTest.cpp
-)
-
-add_dependencies(PiTests sycl)
-target_link_libraries(PiTests PRIVATE sycl LLVMTestingSupport)
\ No newline at end of file
+  )
```

Comment:
```suggestion
)
```

Reply:
I am following the indentation used by other `CMakeLists.txt` files in the LLVM project.

The other often used alternative us to move the `)` to the end of the last content line above it.